test/compilable/test20909.d

```diff
@@ -0,0 +1,16 @@
+// https://issues.dlang.org/show_bug.cgi?id=20909
+/*
+TEST_OUTPUT:
+---
+4$?:32=u|64=LU$
+8$?:32=u|64=LU$
+---
+*/
+
+struct S
+{
+    long a;
+    pragma(msg, x.sizeof);
+    pragma(msg, x.offsetof); // OK now
```

Comment:
is it possible to `static assert` instead of _pragma-messaging_ ?

Reply:
yes